modules/microshift-accessing-cluster-locally.adoc

```diff
@@ -2,6 +2,7 @@
 //
 // microshift/microshift_install/microshift-install-rpm.adoc
 // microshift/microshift_install/microshift-embed-in-rpm-ostree.adoc
+// microshift/microshift_configuring/microshift-kubeconfig.adoc
```

Comment:
Just noting that if you decide to rename the assembly file, you'll also want to update these "included in" comments to match the new name.